release-team/role-handbooks/docs/Release-Timeline.md

```diff
@@ -661,6 +661,11 @@ Enable branch protection on the new `dev-` branch, and deprecate the older one;
 
 Create a milestone for NEW upcoming release. Depending on your permissions, you might need to contact a SIG Docs maintainer. Move anything missed for the current release to the new milestone.
 
+### Modify prow config file
+
+Create a [pull request](https://github.com/kubernetes/test-infra/pull/19877) against [k/test-infra](https://github.com/kubernetes/test-infra)
+ to configure [prow](https://github.com/kubernetes/test-infra/tree/master/prow#) to automatically apply milestones to future release branch
+
```

Comment:
I wasn't sure if I should add a cleanup step to remove the last release branch from the prow config. Can I get some feedback on this?

Reply:
@nikhita could you please advjce us on this?

Reply:
I think it will also be helpful if Sig Docs can comment on how the release branch is used after the release? I see that branch doesn't get deprecated, but does it get used after the release? will prow need to keep applying milestones?

When does milestone get deprecated?

Reply:
> When does milestone get deprecated?

It gets closed after the release and if anything is left in the milestone after the release, it gets moved to the next one or gets closed.

Reply:
If that's the case, then I think we should include clean up in the pull request. Pull request should include clean up of recent release and the setup of a future branch.

Reply:
If it’ll be closed then yes we should include the clean up mechanism as well. Lgtm from me if it’s added 👍

Reply:
updated!